### ISSUE TITLE
transcription error

### ISSUE THREAD

**Issue by saisrinivas047** [facebookresearch/wav2letter#43]
when i transcribed my audio some how the repeated letters in a word is getting replaced by '1'.
example:
original:
two is a whole lot of our analytics around what customers are looking for based on we mine a lot of data from our website on sales searches on what gets requested in branches so we gotten smarter and smarter about the skus that we add meaning that we are adding things that we are pretty certain have some built in demand

wav2letter:
who a whole lot of our own alet around ane part the mo **wil1** for be dano when we mind a want a ban ong that night on **bel1** dirty on on one any weped it an brachet to wen got omalen about that we had **ne1ner1ing** thing that would plas certain have dome built into man

**Comment by ambigus9** [comment]
@vineelpratap It's necessary to make manual transcriptions of any audio file to get an Inference of the same audio?

**Comment by vineelpratap** [comment]
Yes, we currently require it so that we can compute WER. You can keep some dummy values in there for your custom dataset if you don't care about it.

**Comment by ambigus9** [comment]
@vineelpratap I don't care about WER, so I just put a word, for example: "HI" on myaudio.trans.txt file, right? Where I must to save this trans.txt file?
